libs/expo/betterangels/src/lib/screens/AddEditClient/HouseholdMembers/Household/index.tsx

```diff
@@ -0,0 +1,206 @@
+import {
+  Colors,
+  Radiuses,
+  Regex,
+  Spacings,
+} from '@monorepo/expo/shared/static';
+import {
+  DatePicker,
+  Input,
+  Radio,
+  Select,
+  TextBold,
+  TextButton,
+  TextRegular,
+} from '@monorepo/expo/shared/ui-components';
+import { useFormContext } from 'react-hook-form';
+import { View } from 'react-native';
+import {
+  CreateClientProfileInput,
+  GenderEnum,
+  RelationshipTypeEnum,
+  UpdateClientProfileInput,
+} from '../../../../apollo';
+import {
+  enumDisplayGender,
+  enumDisplayRelevant,
+} from '../../../../static/enumDisplayMapping';
+
+interface IHouseholdProps {
+  index: number;
+  remove: (index: number) => void;
+}
+
+export default function Household(props: IHouseholdProps) {
+  const { index, remove } = props;
+  const { control, setValue, watch } = useFormContext<
+    UpdateClientProfileInput | CreateClientProfileInput
+  >();
+
+  const relationship = watch(
+    `householdMembers[${index}].relationshipToClient` as `householdMembers.${number}.relationshipToClient`
+  );
+
+  const householdMembers = watch('householdMembers') || [];
+
+  const handleRemove = () => {
+    remove(index);
+  };
+
+  const handleReset = () => {
+    setValue(
+      `householdMembers[${index}].relationshipToClient` as `householdMembers.${number}.relationshipToClient`,
+      null
+    );
+    setValue(
+      `householdMembers[${index}].name` as `householdMembers.${number}.name`,
+      null
+    );
+    setValue(
+      `householdMembers[${index}].gender` as `householdMembers.${number}.gender`,
+      null
+    );
+    setValue(
+      `householdMembers[${index}].dateOfBirth` as `householdMembers.${number}.dateOfBirth`,
+      null
+    );
+    setValue(
+      `householdMembers[${index}].relationshipToClient` as `householdMembers.${number}.relationshipToClient`,
+      null
+    );
+  };
+
+  if (!relationship) {
+    return (
+      <Select
+        boldLabel
+        labelMarginLeft="xs"
+        label="Type of Relationship"
+        placeholder="Select Relationship"
+        defaultValue={(relationship as RelationshipTypeEnum | undefined) ?? ''}
+        onValueChange={(enumValue) =>
+          setValue(
+            `householdMembers[${index}].relationshipToClient` as `householdMembers.${number}.relationshipToClient`,
+            enumValue as RelationshipTypeEnum
+          )
+        }
+        items={Object.entries(enumDisplayRelevant)
```

Comment:
sorry, I should have noticed this in the last PR. instead of filtering here, can we create a list for each category? maybe something like 

```javascript
// libs/expo/betterangels/src/lib/static/enumDisplayMapping.ts

export const clientRelationEnumDisplay: {
  [key in RelationshipTypeEnum]: string;
} = {
  [RelationshipTypeEnum.CurrentCaseManager]: 'Current Case Manager',
  [RelationshipTypeEnum.Aunt]: 'Aunt',
  [RelationshipTypeEnum.Child]: 'Child',
  [RelationshipTypeEnum.Cousin]: 'Cousin',
  [RelationshipTypeEnum.Father]: 'Father',
  [RelationshipTypeEnum.Friend]: 'Friend',
  [RelationshipTypeEnum.Grandparent]: 'Grandparent',
  [RelationshipTypeEnum.Mother]: 'Mother',
  [RelationshipTypeEnum.Organization]: 'Organization',
  [RelationshipTypeEnum.PastCaseManager]: 'Past Case Manager',
  [RelationshipTypeEnum.Pet]: 'Pet',
  [RelationshipTypeEnum.Sibling]: 'Sibling',
  [RelationshipTypeEnum.Uncle]: 'Uncle',
  [RelationshipTypeEnum.Other]: 'Other',
};

export const clientRelevantContactEnumDisplay: {
  [key in RelationshipTypeEnum]: string;
} = {
  ...clientRelationEnumDisplay,
  [RelationshipTypeEnum.CurrentCaseManager]: 'Current Case Manager',
  [RelationshipTypeEnum.PastCaseManager]: 'Past Case Manager',
  [RelationshipTypeEnum.Organization]: 'Organization',
};

export const clientHouseholdMemberEnumDisplay: {
  [key in RelationshipTypeEnum]: string;
} = {
  ...clientRelationEnumDisplay,
  [RelationshipTypeEnum.Pet]: 'Pet',
};
```

Reply:
done